### PR TITLE
Added C509 paragraph in protocol overview Section 4

### LINKED ISSUE #281: do a quick check that future certificate formats can be supported
In particular, new formats like x509-in-CBOR .  Procedure should be ready to extend to new cert types?

Have a look again at https://datatracker.ietf.org/doc/html/rfc9360 (check how this RFC could apply further than already currently used)

### PR DESCRIPTION
**Pull request by EskoDijk**
also added ref to Pledge discovery section 15 in intro (which was missing). 
Close #281

### CODE REVIEW

**Comment by EskoDijk** [comment]
@toerless Added you here as optional PR reviewer; since we discussed this about 2 weeks ago in the call!

**Comment by toerless** [comment]
Looks good. I would replace "code size savings" with a term like "code sharing benefits with CBOR libraries", but otherwise approved, please merge.

**Comment by toerless** [comment]
Oops. wrong button.

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -224,7 +228,10 @@ as described in {{rpk-considerations}}, the Registrar's Raw Public Key (RPK) is
 For the constrained voucher the default method to indicate ("pin") a trusted domain identity is the domain's PKIX CA certificate,
 but when operating PKIX-less instead the RPK of the Registrar is pinned.
 
-The BRSKI architectures mandates that the MASA be aware of the capabilities of the Pledge.
+For certificates, cBRSKI currently uses the X.509 format, like BRSKI. The protocol and data formats are defined such that future extension to other certificate formats is enabled. For example, CBOR-encoded and COSE-signed 
+C509 certificates ({{I-D.ietf-cose-cbor-encoded-cert}}) may provide data size savings as well as code size savings when applied to cBRSKI.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:232]:
```suggestion
C509 certificates ({{I-D.ietf-cose-cbor-encoded-cert}}) may provide data size savings as well as code sharing benefits with CBOR/COSE libraries, when applied to cBRSKI.
```

**Pull request merged**